src/site/shared/containers/HistoryBox/index.tsx

```diff
@@ -69,21 +100,48 @@ export const HistoryBox = ({ info }: Props) => {
         state => ({ ...state.itemNav })
     );
     const dispatch = useSharedDispatch();
-
+    
     const { undoHistory, redoHistory } = useHistory(info);
 
-    return (
-        <div className={`historybox ${isOpen ? "" : "historybox__move"} ${isHistoryBoxOpen ? "" : "hide"}`}>
-            <div>
-                <span>History</span>
-                <span onClick={() => dispatch(CloseHistoryBox())}>×</span>
+    var redoHasMembers = false;
```

Comment:
nit: use `let` instead of `var`

Reply:
Actually it doesn't look like this variable is used at all, it can probably just be deleted